utm_rally_point[] rally_point_list


